src/main/java/ledger/controller/DbController.java

```diff
@@ -31,6 +31,7 @@
     private List<CallableMethodVoidNoArgs> transactionSuccessEvent;
     private List<CallableMethodVoidNoArgs> accountSuccessEvent;
     private List<CallableMethodVoidNoArgs> payeeSuccessEvent;
+    private List<CallableMethodVoidNoArgs> ignoredExpSuccessEvent;
```

Comment:
Yeah Its at the bottom of the file. Its used in the update of the Ignored Expression table view